rita/src/rita_client/dashboard/network_endpoints.rs

```diff
@@ -370,3 +372,102 @@ pub fn remote_logging_level(path: Path<String>) -> Box<Future<Item = HttpRespons
 
     return Box::new(future::ok(HttpResponse::Ok().json(())));
 }
+
+pub fn add_exits(
+    new_exits: Json<HashMap<String, ExitServer>>,
+) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    debug!("/exits POST hit with {:?}", new_exits);
+    let exits = &mut SETTING.get_exit_client_mut().exits;
+    exits.extend(new_exits.into_inner());
+
+    Box::new(future::ok(HttpResponse::Ok().json(exits.clone())))
+}
+
+pub fn exits_sync(
+    list_url_json: Json<HashMap<String, String>>,
+) -> Box<Future<Item = HttpResponse, Error = Error>> {
+    debug!("/exits/sync hit with {:?}", list_url_json);
+
+    let list_url = match list_url_json.get("url") {
+        Some(url) if url.starts_with("https://") => url,
+        Some(_unsafe_url) => {
```

Comment:
Wouldn't it make sense to match "http://", and another match that would say that the exit url is invalid?

```suggestion
        Some(url) if url.starts_with("http://") => {
```